src/elements/common/sub-header/__tests__/SortButton.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SortButtonBase } from '../SortButton';
+import Button from '../../../../components/button';
+import IconSort from '../../../../icons/general/IconSort';
+
+const intlMock = {
+    formatMessage: jest.fn().mockReturnValue('Sort'),
+};
+
+describe('Elements/SubHeader/SortButton', () => {
+    const getWrapper = () => shallow(<SortButtonBase intl={intlMock} />);
+
+    test('should render IconSort', () => {
+        const wrapper = getWrapper();
+        expect(wrapper.exists(IconSort)).toBe(true);
+    });
+
+    test('should have aria-label "sort"', () => {
+        const wrapper = getWrapper();
+        const button = wrapper.find(Button);
+        expect(button.prop('aria-label')).toBe('Sort');
+        expect(button.prop('aria-describedBy')).toBeFalsy();
```

Comment:
the test above for IconSort uses `.toBe(true)`

might change this to match `.toBe(false)`

Reply:
I dont think that will work - I'm checking for aria-describedBy to be undefined, null, or an empty string, and I think they will fill the `toBe(false)` condition